osbuild/meta.py

```diff
@@ -333,8 +333,32 @@ def __bool__(self):
         },
         "schema_2": {
             "type": "object",
-        }
-    }
+        },
+        "prerequisites": {
+            "type": "object",
+            "additionalProperties": False,
+            "properties": {
+                "buildroot": {
+                    "type": "object",
+                    "additionalProperties": False,
+                    "properties": {
+                        "requires": {
```

Comment:
Isn't this reduandant? To me prerequisites implies requires. Trying to shift this schema left a bit ;)